package.json

```diff
@@ -58,12 +59,13 @@
     "lint:check" : "eslint",
     "lint:check:all" : "eslint lib system_test test",
     "lint:fix" : "eslint --fix",
-    "test": "mocha -timeout 180000 --recursive --full-trace test/unit/**/*.js test/unit/*.js",
-    "test:integration": "mocha -timeout 180000 --recursive --full-trace test/integration/**/*.js test/integration/*.js",
-    "test:single": "mocha -timeout 180000 --full-trace",
-    "test:system": "mocha -timeout 180000 --recursive --full-trace system_test/*.js",
-    "test:unit": "mocha -timeout 180000 --recursive --full-trace test/unit/**/*.js test/unit/*.js",
-    "test:ci": "mocha -timeout 180000 --recursive --full-trace test/**/*.js system_test/*.js"
+    "test": "nyc --silent --no-clean mocha -timeout 180000 --recursive --full-trace test/unit/**/*.js test/unit/*.js",
```

Comment:
we don't want to have nyc running for each test scripts - let's configure it only for some of them e.g. `test:ci:coverage`

Reply:
Ok I've created separated task.